### PR TITLE
update to use the 0.8.0 aihwkit whl

### PR DESCRIPTION
**Pull request by kkvtran**
## Related issues

https://github.com/IBM/aihwkit/issues/535

## Description

Update the notebooks to use the aihwkit-0.8.0+cuda117-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl

### CODE REVIEW

File: notebooks/analog_training_LeNet5.ipynb [hunk 1]
```diff
@@ -118,15 +117,27 @@
...
+      "text/plain": [
+       "tensor([[-0.1216,  0.4219],\n",
+       "        [-0.2063,  0.5724]], grad_fn=<AddBackward0>)"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
```
Comment (maljoras) [inline notebooks/analog_training_LeNet5.ipynb:140]: @kkvtran can you delete your hidden data of the notebook (clear cell outputs)

**Review by maljoras — CHANGES_REQUESTED** [review]
Could you delete the cell outputs of the notbooks before committing ? We had that before, too.  It saves some hidden user data

File: notebooks/analog_training_LeNet5.ipynb (hunk 1 above)
Comment (kkvtran) [reply notebooks/analog_training_LeNet5.ipynb:140]: OK, I can do the kernel restart and remove all the existing output.

**Review by maljoras — APPROVED** [review]
Thanks @kkvtran !

**Pull request merged**